translations/pots/dosomething_miscellaneous.pot

```diff
@@ -95,3 +95,23 @@ msgstr ""
 #: Login error message
 msgid "Password field is required."
 msgstr ""
+
+#: Brand Slogan
+msgid "Let's do this!"
+msgstr ""
+
+#: Brand Slogan
+msgid "Join others making a difference."
+msgstr ""
+
+#: Brand Slogan
+msgid "A global movement for good."
+msgstr ""
+
+#: Brand Slogan
+msgid "Find ways to take action both online and off."
+msgstr ""
+
+#: Brand Slogan
+msgid "Choose a campaign and take action with 5.3 million young people."
+msgstr ""
```

Comment:
it's giving a warning here cause you need to add a carriage return after this line; needs an empty line at bottom of the page. Install the *EditorConfig* SublimeText plugin so that it'll use the `.editorconfig` file in the repo and do this automatically for files :)